src/main/java/org/dataconservancy/cos/osf/client/model/Wiki.java

```diff
@@ -0,0 +1,214 @@
+import org.dataconservancy.cos.osf.client.support.DownloadLinkTransform;
+import org.dataconservancy.cos.osf.client.support.FileIdTransform;
+import org.dataconservancy.cos.osf.client.support.JodaSupport;
+import org.dataconservancy.cos.osf.client.support.ProviderIdTransform;
+import org.dataconservancy.cos.rdf.annotations.IndividualUri;
+import org.dataconservancy.cos.rdf.annotations.OwlIndividual;
+import org.dataconservancy.cos.rdf.annotations.OwlProperty;
+import org.dataconservancy.cos.rdf.support.OwlClasses;
+import org.dataconservancy.cos.rdf.support.OwlProperties;
+import org.joda.time.DateTime;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.github.jasminb.jsonapi.RelType;
+import com.github.jasminb.jsonapi.ResolutionStrategy;
+import com.github.jasminb.jsonapi.annotations.Id;
+import com.github.jasminb.jsonapi.annotations.Links;
+import com.github.jasminb.jsonapi.annotations.Relationship;
+import com.github.jasminb.jsonapi.annotations.Type;
+
+
+/**
+ * Wiki model for OSF
+ * Created by Ben Trumbore on 9/16/2016.
+ */
+//@Type("wiki")
+//@JsonIgnoreProperties(ignoreUnknown = true)
+//@OwlIndividual(OwlClasses.OSF_WIKI)
+public class Wiki {
+
+    /*unique OSF ID for the file**/
+    //@Id
+    //@IndividualUri(transform = ProviderIdTransform.class)
+    private String id;
+
+    /**list of comments associated with wiki*/
+    //@Relationship(value = "comments", resolve = true, relType = RelType.RELATED, strategy = ResolutionStrategy.OBJECT)
+    private List<Comment> comments;
+
+    /**Gets other links found in data.links:{} section of JSON**/
+    //@Links
+    //@OwlProperty(value = OwlProperties.OSF_HAS_BINARYURI, transform = DownloadLinkTransform.class)
+    Map<String, ?> links;
+
+    /**name of the wiki; used for display*/
+    //@OwlProperty(OwlProperties.OSF_HAS_NAME)
+    private String name;
+
+    /**"file" or "folder"*/
+    //@OwlProperty(OwlProperties.OSF_HAS_HASKIND)
+    private String kind;
+
+    /**the unix-style path to the file relative to the provider root*/
+    //@OwlProperty(OwlProperties.OSF_HAS_MATERIALIZEDPATH)
+    private String materialized_path;
+
+    /**timestamp of when this wiki was last updated*/
+    //@OwlProperty(value = OwlProperties.OSF_HAS_DATEMODIFIED, transform = DateTimeTransform.class)
+    private DateTime date_modified;
+
+    /**node this wiki belongs to*/
+    //@OwlProperty(OwlProperties.OSF_HAS_NODE)
+    private String node;
+
+    /**user that owns this wiki*/
+    //@OwlProperty(OwlProperties.OSF_HAS_NODE)
+    private String user;
+
+    /**same as for corresponding WaterButler entity*/
+    //@OwlProperty(OwlProperties.OSF_HAS_PATH)
+    private String path;
+
+    /**  */
+    //@OwlProperty(OwlProperties.OSF_HAS_PATH)
+    private String content_type;
+
+    /**version of wiki*/
+    //@OwlProperty(OwlProperties.OSF_HAS_SIZE)
+    private Integer version;
```

Comment:
We may need to have a field for `extra`, which would contain the `version` property, rather than have `version` on its own.